src/actuator/ActuatorBulkFAST.C

```diff
@@ -62,18 +62,23 @@ ActuatorBulkFAST::ActuatorBulkFAST(
 
 ActuatorBulkFAST::~ActuatorBulkFAST() { openFast_.end(); }
 
+bool
+ActuatorBulkFAST::is_tstep_ratio_admissable(
+  const ActuatorMetaFAST& actMeta, const double naluTimeStep)
+{
+  const double stepCheck =
+    std::abs(naluTimeStep / actMeta.fastInputs_.dtFAST - tStepRatio_);
```

Comment:
`actMeta.fastInputs_.dtFAST` is out of sync with `fast::OpenFAST::dtFAST`

Reply:
@sayerhs I'm not seeing a way to get `dtFAST` from `fast::OpenFAST`.  Looks like it's a private member with no getter function.  I'm going to just comment this new check out and open an issue with OpenFAST to add that feature in.